src/bitmap/store.rs

```diff
@@ -43,14 +43,18 @@ impl Store {
         }
     }
 
+    /// Push the value that must be the new max of the set.
+    /// This function returns whether the value is equal to the
+    /// last max. This infomation is needed to correctly update the
+    /// length of the container.
```

Comment:
Just to have a better rendering of the doc :)

```suggestion
    /// Push the value that must be the new max of the set.
    ///
    /// This function returns whether the value is equal to the
    /// last max. This infomation is needed to correctly update the
    /// length of the container.
```